package.json

```diff
@@ -38,17 +38,17 @@
     "@expo/vector-icons": "14.0.0",
     "@native-html/iframe-plugin": "^2.6.1",
     "@native-html/table-plugin": "^5.3.1",
-    "@react-native-async-storage/async-storage": "1.21.0",
-    "@react-native-community/datetimepicker": "7.7.0",
-    "@react-native-community/netinfo": "11.1.0",
-    "@react-native-masked-view/masked-view": "0.3.0",
+    "@react-native-async-storage/async-storage": "1.23.1",
+    "@react-native-community/datetimepicker": "8.0.1",
+    "@react-native-community/netinfo": "11.3.1",
+    "@react-native-masked-view/masked-view": "0.3.1",
     "@react-navigation/bottom-tabs": "^6.5.9",
     "@react-navigation/drawer": "^6.6.4",
     "@react-navigation/elements": "^1.3.19",
     "@react-navigation/native": "^6.1.8",
     "@react-navigation/stack": "^6.3.18",
-    "@sentry/react-native": "~5.20.0",
-    "@shopify/flash-list": "1.6.3",
+    "@sentry/react-native": "~5.22.0",
```

Comment:
As far as I learnt, the sentry-expo package is no longer used since SDK 50. But we continue to use the sentry-expo package in the app. According to the documentation, @sentry/react-native package is recommended. Do you have any information about this? Maybe expo-doctor's error may be caused by this.

https://github.com/expo/sentry-expo

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1142/commits/981466e38a85970b1c6dbfe1ab31b0b464acebe8

Reply:
good catch!